python/tvm/auto_scheduler/loop_state.py

```diff
@@ -119,9 +137,61 @@ def reorder(self, stage, order):
         order : List[Iterator]
             Iterators in the expected order.
         """
-        stage_id = self._resolve_stage_id(stage)
+        self.state_object = _ffi_api.StateReorder(self.state_object, self._resolve_stage_id(stage),
+                                                  order)
+
+    def compute_at(self, stage, target_stage, target_iter):
+        """ Schedule primitive corresponds to te.compute_at.
+
+        Parameters
+        ----------
+        stage : Union[int, Operation, Tensor]
+            The Stage to be compute at, can be a Stage order index, Stage operation or stage
+            output tensor.
+        target_stage : Union[int, Operation, Tensor]
+            The target stage of compute_at, can be a Stage order index, Stage operation or stage
+            output tensor.
+        target_iter : Iterator
+            The target Iterator of compute_at.
+
+        Notes
+        -----
+        After compute_at, the extent of each iterator may not be accurate any more, so the bound
+        information will be removed from this state. Run ComputeDAG::InferBound to recover.
```

Comment:
```suggestion
        After compute_at, we need careful dependency analysis to compute the accurate bound information.
        However, it is relatively expensive and complicated. So in LoopState, we just fill "None" as bound 
        for the newly created iterators. If you do need the bound, you can call ComputeDAG::InferBound on
        the returned state to get all bound information.
```

Please propagate this change.